apps/tailwind-components/pages/input/Int.story.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <FormFields
```

Comment:
You don't really need this separate stories anymore. We have the 'FormFields' story and that should have all types.

Can you at least add these types to the FormField types.